crates/pessimistic-proof-core/src/aggchain_data/aggchain_hash.rs

```diff
@@ -1,85 +1,67 @@
 use agglayer_primitives::{
     bytes::{BigEndian, ByteOrder as _},
     keccak::keccak256_combine,
-    Address, Digest,
+    Digest,
 };
+use hex_literal::hex;
 
-use crate::aggchain_data::{aggchain_proof::AggchainProof, AggchainData, Vkey};
+use crate::aggchain_data::{aggchain_proof::AggchainProof, AggchainData, MultiSignature, Vkey};
 
 struct ConsensusType(u32);
 
 pub enum AggchainHashValues {
-    ConsensusType0 {
-        signer: Address,
-    },
     ConsensusType1 {
         aggchain_vkey: Option<Vkey>,
         aggchain_params: Option<Digest>,
-        signers_commit: Option<Digest>,
-        threshold: Option<usize>,
+        multisig_hash: Digest,
     },
 }
 
 impl From<&AggchainHashValues> for ConsensusType {
     fn from(value: &AggchainHashValues) -> Self {
         match value {
-            AggchainHashValues::ConsensusType0 { .. } => Self(0),
             AggchainHashValues::ConsensusType1 { .. } => Self(1),
         }
     }
 }
 
 impl AggchainHashValues {
-    /// Returns the commitment on signers in case of no signer.
-    pub fn empty_signers() -> Digest {
-        keccak256_combine([Digest::default()])
-    }
+    /// Value if no multisig.
+    pub const EMPTY_MULTISIG_HASH: Digest = Digest(hex!(
+        "290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563"
+    ));
 
-    /// Returns the empty vkey used in case of no aggchain proof
-    pub fn empty_sp1_vkey() -> Vkey {
-        Vkey::default()
-    }
+    /// Value if no aggchain vkey.
+    pub const EMPTY_AGGCHAIN_VKEY: Digest = Digest::ZERO;
```

Comment:
```suggestion
    pub const EMPTY_AGGCHAIN_VKEY_HASH: Digest = Digest::ZERO;
```

Reply:
Just to be consistent - on one place we refer to it as `aggchain_vkey_hash`, on another as aggchain_vkey